app/services/analytics_events.rb

```diff
@@ -636,8 +636,11 @@ def idv_forgot_password_confirmed
   end
 
   # GPO address letter requested
-  def idv_gpo_address_letter_requested
-    track_event('IdV: USPS address letter requested')
+  def idv_gpo_address_letter_requested(enqueued_at:)
+    track_event(
```

Comment:
It is helpful to have this logged here. We'll also need it on the events where the user enters the code. Those are all triggered in this controller: https://github.com/18F/identity-idp/blob/main/app/controllers/idv/gpo_verify_controller.rb